starburst-presto/presto.sh

```diff
@@ -23,7 +23,7 @@ readonly PRESTO_MASTER_FQDN="$(/usr/share/google/get_metadata_value attributes/d
 readonly WORKER_COUNT=$(/usr/share/google/get_metadata_value attributes/dataproc-worker-count)
 readonly PRESTO_MAJOR_VERSION="302"
 readonly STARBURST_PRESTO_VERSION="302-e.11"
-readonly HTTP_PORT="8080"
+readonly HTTP_PORT="8070"
```

Comment:
If we're going to change from Presto's default port, I would suggestion port 8060 to be consistent with: https://cloud.google.com/dataproc/docs/tutorials/presto-dataproc
> If using the optional component, the Presto server will run on port 8060.

Reply:
Okay Updated :)

Reply:
@ChethanUK what is the problem with 8080 port?
This is the default Presto port. Also, init action for Github Presto is also using 8080:
https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/presto/presto.sh#L31

Reply:
No, 8080 as another service, if we use that we can't use other services like Apache Zepplin, etc.. So, It best to have 8060 or some other port other than 8080